src/frontends/tensorflow/src/op/approximate_equal_op.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+#include "openvino/op/abs.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/subtract.hpp"
+
+using namespace std;
+using namespace ov::op;
+using namespace ov::frontend::tensorflow;
```

Comment:
please move this file into `src\frontends\tensorflow_common\src\op`. The reason you had to add `using namespace ov::frontend::tensorflow;` is a wrong directory for this file.
Revert your last commit and move to the right directory.
It is needed because `tensorflow_common` contains translators for both TF and TF Lite models. So we can re-use your `translate_approximate_equal_op` for TF Lite latter.

Best regards,
Roman

Reply:
@rkazants thank you for your input. I have addressed the changes please give it a look .
Regards, 
Monal